api/service/v1alpha1/service.proto

```diff
@@ -68,13 +69,18 @@ service KargoService {
   rpc RefreshWarehouse(RefreshWarehouseRequest) returns (RefreshWarehouseResponse);
 
   /* Credential APIs */
+
   rpc CreateCredentials(CreateCredentialsRequest) returns (CreateCredentialsResponse);
   rpc DeleteCredentials(DeleteCredentialsRequest) returns (DeleteCredentialsResponse);
   rpc GetCredentials(GetCredentialsRequest) returns (GetCredentialsResponse);
   rpc ListCredentials(ListCredentialsRequest) returns (ListCredentialsResponse);
   rpc UpdateCredentials(UpdateCredentialsRequest) returns (UpdateCredentialsResponse);
 
   /* AnalysisRun APIs */
```

Comment:
Nit: Since these endpoints are for both runs and templates now, we could change this:

```suggestion
  /* Analysis APIs */
```